### PR TITLE
doc: add minutes for 15th Nov 2022

### PR DESCRIPTION
**Pull request by gireeshpunathil**
Fixes: https://github.com/nodejs/diagnostics/issues/598

@nodejs/diagnostics

### CODE REVIEW

File: wg-meetings/2022-11-15.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Agenda
+
+## Announcements
+
+*Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/diagnostics
+
+* llnode - Current Status  [#576](https://github.com/nodejs/diagnostics/issues/576)
+  * Tony, some work on documentation, no other update
```
Comment (tony-go) [inline wg-meetings/2022-11-15.md:27]:
```suggestion
  * Tony: I haven't time to work on the documentation issue, there is no other update
```

**Review by tony-go — COMMENTED** [review]
one little change ^^

**Review by tony-go — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by gireeshpunathil** [comment]
landed as 0d04ec69457931f686c51262fe7cade72b758bd6

**Pull request closed without merging**